functions.ts

```diff
@@ -98,4 +98,21 @@ export const functions: Array<
       },
     },
   },
+  // Run code
```

Comment:
this is probably too much general, either you want run just shell commands or python code, each should have different instructions. Otherwise the model can produce input which won't be compatible. E.g. generating just code for python without `python -c` before the code.

Reply:
it just worked for me, I was able to run python, github cli. even I am thinking on simplifing the original tools that you made, becuase it can just ran this like "run a github command to merge and rebase pull requests". Or make a github tool that does anything requested through the github cli

Reply:
but probably i will iterate with this later